README.md

```diff
@@ -153,8 +155,13 @@ duplicate([1,2,3,4,5]); // [1,2,3,4,5,1,2,3,4,5]
 * What is event loop?
   * What is the difference between call stack and task queue?
 * Explain the differences on the usage of `foo` between `function foo() {}` and `var foo = function() {}`
+* What is object-oriented JavaScript (OOJS)?
+* What are teh 6 primitive data types in JavaSript?
```

Comment:
not sure if we need to give hint about `6`

Reply:
ok, should I take out the # 6?